### DISCUSSION TITLE
Train floret embedding iteratively

### DISCUSSION THREAD

**Discussion by mvivopas** [explosion/spaCy#11414] [Help: Model Advice] [answered]
I'm trying to train a word embedding model using [floret embeddings](https://github.com/explosion/floret). However I have the constraint I have to train the model by batches of data (batches are big ~1M notes/batch). I have already trained a Word2Vec model using Gensim. Now I would like to train a floret model iteratively, but I haven't found any documentation neither the code in the [repository](https://github.com/explosion/floret) seems to let train the model from pre-trained embeddings.

**_Could someone help me find out if there is a way of training iteratively floret embeddings?_**

**Comment by adrianeboyd** [comment, marked as answer]
There currently isn't a way to do this. The original fastText code doesn't support this, but you can find some related issues and open PRs with more info in the fastText github repo.
